contents/docs/cdp/destinations/advanced.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Destinations: advanced usage"
```

Comment:
I'm not 100% sure about this title, maybe:

- Customizing destinations
- Building custom destinations
- Modifying destinations

It is a bit unclear what "advanced" means from the outside and someone looking to build or modify a destination won't immediately think of it as "advanced"

Reply:
Customizing sounds right, because we're running the spectrum between explaining templating all the way down to writing your own Hog